### ISSUE TITLE
Corrupt manifest.json?

### ISSUE THREAD

**Issue by PapiDimmi** [Jessecar96/SteamDesktopAuthenticator#371]
It worked a couple weeks ago, but now, it just doesn’t open. There is no error or anything; the program won’t open at all.

When I launch SDA, it does appear in Task Manager, but I can’t access the program. After a few seconds, it disappears from Task Manager. I’ve tried updating Windows, rebooting, changing UAC settings, and running SDA as an administrator, but nothing works.

I use Windows 10. I have seen other people with the same issue as I have, but none of them has been solved. I haven’t updated SDA itself, so perhaps a Windows update messed with something?

I need SDA to trade or sell items on the Steam Community Market, so it’s urgent for this issue to be solved for me. Could anyone please help me?

**Comment by PapiDimmi** [comment]
I can’t trade or use the market without SDA, so… can anyone please help me? I’d really appreciate it.

**Comment by PapiDimmi** [comment]
I deleted the “maFiles” folder, and the program opened. I can’t, however, import an old folder or try to log in with my Steam account anew, as I get [this error](https://pastebin.com/raw/3L30ExeN).

I also tried with the newest version of SDA, but the same thing happened. Please help me.

**Comment by PapiDimmi** [comment]
After searching for other issues, I found #252.

I checked my manifest.json file, and it has a bunch of blank characters.

Someone said that I have to remove the mobile authenticator and start anew, since I encrypted it.

***OH LORD***, I _cannot_ wait 15 more days. Is there _anything_ I can do? Oh my god, I have already waited 15 days. I can’t bear to wait 15 more. What the hell?! Why did this happen?

**Comment by PapiDimmi** [comment]
After searching for other issues, I found #252.

I checked my manifest.json file, and it has a bunch of blank characters.

Someone said that I have to remove the mobile authenticator and start anew, since I encrypted it.

***OH LORD***, I _cannot_ wait 15 more days. Is there _anything_ I can do? Oh my god, I have already waited 15 days. I can’t bear to wait 15 more. What the hell?! Why did this happen?

I will do _anything_ if someone can help me fix this.

**Comment by PapiDimmi** [comment]
After searching for other issues, I found #252.

I checked my manifest.json file, and it has a bunch of blank characters.

@hyt47 said that I have to remove the mobile authenticator and start anew, since I encrypted it.

***OH LORD***, I _cannot_ wait 15 more days. Is there _anything_ I can do? Oh my god, I have already waited 15 days. I can’t bear to wait 15 more. What the hell?! Why did this happen?

I will do _anything_ if someone can help me fix this.
